lib/utils.js

```diff
@@ -233,11 +234,11 @@ async function overQuota (root, serverUri) {
  * so it needs to be rewritten.
  */
 
-function actualSize (root) {
+async function actualSize (root) {
   return util.promisify(getSize)(root)
 }
 
-function _asyncReadfile (filename) {
+async function _asyncReadfile (filename) {
```

Comment:
I don't think this `async` is needed here, as `promisify` should already return a promise.

Reply:
Fixed in b79bd563d3c582b6babb7018149a55e45e53ebb6